lib/fluent/plugin/filter_record_transformer.rb

```diff
@@ -60,8 +60,8 @@ def configure(conf)
       end
 
       placeholder_expander_params = {
-        :log           => log,
-        :auto_typecast => @auto_typecast,
+        log: log,
+        auto_typecast: @auto_typecast,
```

Comment:
it has just 2 fields and these are far different from each other. There seems no problem.